src/theme/HntToDcSimulator.jsx

```diff
@@ -0,0 +1,107 @@
+import { PythConnection } from '@pythnetwork/client'
+import { Connection, PublicKey } from '@solana/web3.js'
+import React, { useEffect, useState } from 'react'
+import styles from "./../css/HntToDcSimulator.module.css"
+
+const DC_PRICE = 0.00001;
+const RPC_ENDPOINT = 'https://api.devnet.solana.com';
+const PROGRAM_KEY = 'gSbePebfvPy7tRqimPoVecS2UsBvYv46ynrzWocc92s';
+
+const calculateDc = (price) => price / DC_PRICE;
+
+export const HntToDcSimulator = () => {
+  const [liveHntPrice, setLiveHntPrice] = useState(0)
+  const [simulatedHntPrice, setSimulatedHntPrice] = useState(1)
+  const [sliderRange, setSliderRange] = useState({ min: 0, max: 5 })
+  const isInitialPriceSet = !!liveHntPrice
+  const dcAmount = calculateDc(simulatedHntPrice)
+
+  useEffect(() => {
+    const connection = new Connection(RPC_ENDPOINT)
+    const programKey = new PublicKey(PROGRAM_KEY)
+
+    const pythConnection = new PythConnection(connection, programKey)
+    pythConnection.onPriceChange((product, price) => {
+      if (product.symbol === 'Crypto.HNT/USD' && price.price) {
+        setLiveHntPrice(price.price)
+        if (!isInitialPriceSet) {
+          setInitialSliderValues(price.price)
+        }
+      }
+    })
+
+    pythConnection.start()
+    return () => pythConnection.stop()
+  }, [isInitialPriceSet])
+
+  const setInitialSliderValues = (price) => {
+    // based on queried price, update the initial position and min/max of the input
+    const roundedPrice = Math.round(price * 100) / 100
+    setSimulatedHntPrice(roundedPrice)
+    const upperLimit = Math.ceil(price / 5) * 5 + 5
+    let lowerLimit = Math.max(0, Math.floor(price / 5) * 5)
```

Comment:
nit: do we care to have lower limit increase with price? 
On the flip side we are doing 0.01 increments so that is a lot of steps... but its not like its hard to just drag. 🤷

Reply:
Test it with a range of 0-100. It doesn't feel all that bad having such a large range. 
But if you want to keep it as is its also not a hill I'm looking to die on.